content/post/adding_context_with_annotations.md

```diff
@@ -0,0 +1,106 @@
+---
+author: Michelle Ballen
+title: "Adding Context to Your Analysis with Annotations"
+cover: ""
+publishDate: "2018-07-31"
+draft: true
+tags:
+ - documentation
+ - annotations
+---
+
+Anyone who has worked in digital analytics will tell you that day over day performance can be volatile. Shifts in marketing mix can cause fluctuating e-commerce conversion rates, new feature launches can lead to sudden and temporary swings in engagement rates and onsite bugs can result in anomalies in abandonment rates. Some of these scenarios can be diagnosed through extensive segmentation of data. Others, [like a dropped analytics snippet] (https://www.locallyoptimistic.com/post/web-analytics-process/) or a bug with your payment processor, cannot be so easily uncovered. The simplest thing to do when events like these take place is to take a mental note and count on your memory for when you inevitably have to revisit that data in the future. As you can imagine, taking a mental note isn’t a scalable solution for a number of reasons.
+
+It may seem like an ‘unsexy’ practice relative to other data processes, but from my experience, keeping a record of the online and offline events that affect your business is a practice and well worth the (small) time investment.
+
+<!--more-->
+
+#### Why taking a mental note isn’t a scalable solution:
```

Comment:
Suggestion:

Change the section heading to

> #### Examples of Common Failures

And then rephrase the bullets to emphasize the failure point (your second bullet is really good here).